### ISSUE TITLE
"VIew In Github" button no longer in card for Github Enterprise connector

### ISSUE THREAD

**Issue by albertptluo** [integrations/microsoft-teams#357]
**Describe the bug**
Button linking to issue in Github no longer appears on the card
Our team is using the GitHub Enterprise connector in MS Teams, where we have a dedicated channel for GitHub Pull Requests. 
The connector cards used to have a button linking back to Github, but after moving to the new MS Teams, that button has disappeared. 

**Screenshots**
![f860e460-756e-4bb6-aa7a-86129f19b1a4](https://github.com/integrations/microsoft-teams/assets/3270515/15bf2860-6586-4f5b-9826-5f297d4f3ae1)

**Desktop (please complete the following information):**
MS Teams Desktop App for MacOSX
Version 24004.1304.2655.7488

**Comment by MartinWallgren** [comment]
We see the same behavior. Since Teams 2.0 the button is not visible anymore.

**Comment by bwaang** [comment]
Yes, I'm seeing this issue as well - since we have requirements for merge in on PRs, that was effectively the only button I've used...

I did notice in the code that the `<div>` still exists, however it seems like Teams is only displaying the first 3 buttons (probably some new CSS styling?) even if we just shift this button to the 1st one as a hack to get it to appear again, I'd be okay with that for now (but probably other people use the other buttons, so not a long term solution)

It was okay for awhile in `v1` but looks like it's both missing on `v1` & `v2` of teams...

**Comment by sivakumarvunnam1** [comment]
Yes, I've observed the same issue. The button has become invisible since Teams 2.0.

**Comment by troy-davis** [comment]
Also seeing the same problem here. Please fix, this was so convenient.

**Comment by kevinwongtyl** [comment]
Yes please fix!  This is adding friction to our code review process.  Also on Teams 2.0

**Comment by sivakumar-shanmugasu** [comment]
I see same issue also. It would be very useful for our team developers to fix this.

**Comment by MikeGitUser** [comment]
Please fix as soon as you can!  This is really a pain to have to access manually.  :(

**Comment by krzd** [comment]
The card design has changed since and addresses it:
![Image](https://github.com/user-attachments/assets/0d35f8fc-03a0-4b16-a160-37f8a05e25f1)
